karma.conf.js

```diff
@@ -1,78 +1,79 @@
 const { argv } = require('yargs')
+
 const config = require('./config')
 const webpackConfig = require('./webpack.config')
 
-module.exports = (karmaConfig) => {
-  karmaConfig.set({
-    basePath: process.cwd(),
-    browsers: ['PhantomJS'],
-    singleRun: !argv.watch,
-    reporters: ['mocha', 'coverage'],
-    files: [
-      './test/tests.bundle.js',
-    ],
-    formatError(msg) {
-      let haveSeenStack = false
-      return msg
-        .split('\n')
-        .reduce((list, line) => {
-          // filter out node_modules
-          if (/~/.test(line)) return list
+const formatError = (msg) => {
+  let haveSeenStack = false
+  return msg
+    .split('\n')
```

Comment:
This formatter is also borked and I'd like to clean it up with this PR.  It used to receive the full stack and filter out blackboxed scripts, however, some time ago it began receiving each line of the stack trace instead.  This is why the assertions have had so much extra whitespace in them.

I'll push a fix for this.